src/Bridges/Constraint/bridges/norm_infinity.jl

```diff
@@ -31,6 +31,14 @@ end
 const NormInfinity{T,OT<:MOI.ModelLike} =
     SingleBridgeOptimizer{NormInfinityBridge{T},OT}
 
+function MOI.supports_constraint(
+    ::Type{<:NormInfinityBridge{T}},
+    ::Type{MOI.VectorAffineFunction{Complex{T}}},
+    ::Type{MOI.NormInfinityCone},
+) where {T}
+    return false
+end
```

Comment:
Our reoccurring issue is that `supports_constraint` is too lenient. It does _not_ support all of the things that it currently returns `true` for.